src/internal/operations/test/helper_test.go

```diff
@@ -575,14 +581,21 @@ type ids struct {
 	DriveRootFolderID string
 }
 
+type gids struct {
+	ID                        string
+	RootSiteID                string
+	RootSiteDriveID           string
+	RootSiteDriveRootFolderID string
+}
+
```

Comment:
Kinda wish I could recursively nest a struct in itself, because ids containing `RootSite ids` would work great here.  But I think this is a good enough halfway stop.

Mostly suggesting the change as a Nit to ensure we have separate declarations for "what ids we want" vers contextual siloing/subownership of ids.
```suggestion
type gids struct {
        ids
	RootSite ids
}

```

Reply:
I've changed it to the following. Didn't quite understand the use of using ids in the root level.

``` go
type gids struct {
	ID       string
	RootSite ids
}
```